editor/src/components/navigator/navigator-item/navigator-item.tsx

```diff
@@ -21,6 +21,9 @@ import { MetadataUtils } from '../../../core/model/element-metadata-utils'
 import { ThemeObject } from '../../../uuiui/styles/theme/theme-helpers'
 import { isFeatureEnabled } from '../../../utils/feature-switches'
 
+export const NavigatorItemTestId = (pathString: string): string =>
```

Comment:
why not just using the other one?

https://github.com/concrete-utopia/utopia/blob/35a60ea87ef8826598a365eb6a0ca62eb3dee85c/editor/src/components/navigator/navigator-item/navigator-item-dnd-container.tsx#L42

If there's a specific reason not to, can we rename this? the double export for two things that are veeeeery similar is a bit clunky imho